gromacs/compat/__init__.py

```diff
@@ -0,0 +1,345 @@
+"""
+compat
```

Comment:
Also, did you write this or does it come from elsewhere? If so, licenses and attribution is needed.